dev/flutter/todo/submissions/Dishal Kanishka/todo_application/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,33 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <application
+        android:label="todo_application"
```

Comment:
Let's change the label here to a plain english text and add in a suitable app icon without the default flutter app icon